### ISSUE TITLE
track menu state

### ISSUE THREAD

**Issue by no-stack-dub-sack** [no-stack-dub-sack/cs-playground-react#63]
this can be especially useful if number of challenges eventually grows significantly. Will let the user reopen the page and menu will still be folded/not folded as it was last time they visited.

**Comment by mansisce** [comment]
I am picking this up to start with my contribution. Please let me know if this the right piece to pick up.

**Comment by no-stack-dub-sack** [comment]
@mansisce This is the perfect place to get started. If you have familiarity with React and Redux and have a clear idea on how to proceed, I would say go for it! I would really appreciate the help :-)

Let me know if you have any questions or anything. I'm happy to help provide some guidance if need be. Thanks for your interest in helping!

**Comment by mansisce** [comment]
#66 Please find the pull request for the same

**Comment by no-stack-dub-sack** [comment]
@mansisce this issue will be closed once staging is merged into master 😄 

Thanks again for your hard work on this!
